### ISSUE TITLE
compiled on Linux, but can't compile binary

### ISSUE THREAD

**Issue by rcampbel3** [geekuillaume/soundsync#9]
My system is running Ubuntu 14.04 (yeah, I know it's old, but that doesn't appear to be my issue)

Following build from source instructions:

```
rcampbel@Godzilla~/project/soundsync/src/utils/opus_vendor $ ./autogen.sh 
Updating build configuration files, please wait....
libtoolize: putting auxiliary files in `.'.
libtoolize: linking file `./ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIR, `m4'.
libtoolize: linking file `m4/libtool.m4'
libtoolize: linking file `m4/ltoptions.m4'
libtoolize: linking file `m4/ltsugar.m4'
libtoolize: linking file `m4/ltversion.m4'
libtoolize: linking file `m4/lt~obsolete.m4'
Makefile.am:317: warning: '%'-style pattern rules are a GNU make extension
Makefile.am:320: warning: '%'-style pattern rules are a GNU make extension
```
emconfigure ./configure --disable-extra-programs --disable-doc --disable-intrinsics --disable-hardening --disable-rtcd --disable-s
tack-protector  <-- this line in readme.md should have **emconfigure** removed from the front of it

```
rcampbel@Godzilla~/project/soundsync/src/utils/opus_vendor $ ./configure --disable-extra-programs --disable-doc --disable-intrinsics --disable-hardening --disable-rtcd --disable-s
configure: WARNING: unrecognized options: --disable-s
checking whether make supports nested variables... yes
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether to enable maintainer-specific portions of Makefiles... yes
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking how to print strings... printf
checking for style of include used by make... GNU
checking for gcc... gcc-9
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc-9 accepts -g... yes
checking for gcc-9 option to accept ISO C89... none needed
checking whether gcc-9 understands -c and -o together... yes
checking dependency style of gcc-9... gcc3
checking for a sed that does not truncate output... /bin/sed
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for fgrep... /bin/grep -F
checking for ld used by gcc-9... /usr/bin/ld
checking if the linker (/usr/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
checking the name lister (/usr/bin/nm -B) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 1572864
checking whether the shell understands some XSI constructs... yes
checking whether the shell understands "+="... yes
checking how to convert x86_64-unknown-linux-gnu file names to x86_64-unknown-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-unknown-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /usr/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for ar... ar
checking for archiver @FILE support... @
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /usr/bin/nm -B output from gcc-9 object... ok
checking for sysroot... no
checking for mt... mt
checking if mt is a manifest tool... no
checking how to run the C preprocessor... gcc-9 -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc-9 supports -fno-rtti -fno-exceptions... no
checking for gcc-9 option to produce PIC... -fPIC -DPIC
checking if gcc-9 PIC flag -fPIC -DPIC works... yes
checking if gcc-9 static flag -static works... yes
checking if gcc-9 supports -c -o file.o... yes
checking if gcc-9 supports -c -o file.o... (cached) yes
checking whether the gcc-9 linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... yes
checking for gcc-9 option to accept ISO C99... none needed
checking for an ANSI C-conforming const... yes
checking for inline... inline
checking dependency style of gcc-9... gcc3
checking for C/C++ restrict keyword... __restrict
checking for C99 variable-size arrays... yes
checking for cos in -lm... yes
checking if gcc-9 supports -fvisibility=hidden... yes
checking if gcc-9 supports -fstack-protector-strong... yes
checking whether to add -D_FORTIFY_SOURCE=2 to CFLAGS... no
checking if gcc-9 supports -Wall -Wextra -Wcast-align -Wnested-externs -Wshadow -Wstrict-prototypes... yes
checking for lrintf... yes
checking for lrint... yes
checking for __malloc_hook... yes
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating opus.pc
config.status: creating opus-uninstalled.pc
config.status: creating celt/arm/armopts.s
config.status: creating doc/Makefile
config.status: creating doc/Doxyfile
config.status: creating config.h
config.status: executing depfiles commands
config.status: executing libtool commands
configure: WARNING: unrecognized options: --disable-s
configure:
------------------------------------------------------------------------
  opus unknown:  Automatic configuration OK.

    Compiler support:

      C99 var arrays: ................ yes
      C99 lrintf: .................... yes
      Use alloca: .................... no (using var arrays)

    General configuration:

      Floating point support: ........ yes
      Fast float approximations: ..... no
      Fixed point debugging: ......... no
      Inline Assembly Optimizations: . No inline ASM for your platform, please send patches
      External Assembly Optimizations: 
      Intrinsics Optimizations: ...... no
      Run-time CPU detection: ........ disabled
      Custom modes: .................. no
      Assertion checking: ............ no
      Hardening: ..................... no
      Fuzzing: ....................... no
      Check ASM: ..................... no

      API documentation: ............. no
      Extra programs: ................ no
------------------------------------------------------------------------

 Type "make; make install" to compile and install
 Type "make check" to run the test suite
```

then make...

```
rcampbel@Godzilla~/project/soundsync/src/utils/opus_vendor $ make
make  all-recursive
make[1]: Entering directory '/media/rcampbel/4TB_EXT/rcampbel/project/soundsync/src/utils/opus_vendor'
make[2]: Entering directory '/media/rcampbel/4TB_EXT/rcampbel/project/soundsync/src/utils/opus_vendor'
  CC       celt/bands.lo
  CC       celt/celt.lo
  CC       celt/celt_encoder.lo
  CC       celt/celt_decoder.lo
  CC       celt/cwrs.lo
  CC       celt/entcode.lo
  CC       celt/entdec.lo
  CC       celt/entenc.lo
  CC       celt/kiss_fft.lo
  CC       celt/laplace.lo
  CC       celt/mathops.lo
  CC       celt/mdct.lo
  CC       celt/modes.lo
  CC       celt/pitch.lo
  CC       celt/celt_lpc.lo
  CC       celt/quant_bands.lo
  CC       celt/rate.lo
  CC       celt/vq.lo
  CC       silk/CNG.lo
  CC       silk/code_signs.lo
  CC       silk/init_decoder.lo
  CC       silk/decode_core.lo
  CC       silk/decode_frame.lo
  CC       silk/decode_parameters.lo
  CC       silk/decode_indices.lo
  CC       silk/decode_pulses.lo
  CC       silk/decoder_set_fs.lo
  CC       silk/dec_API.lo
  CC       silk/enc_API.lo
  CC       silk/encode_indices.lo
  CC       silk/encode_pulses.lo
  CC       silk/gain_quant.lo
  CC       silk/interpolate.lo
  CC       silk/LP_variable_cutoff.lo
  CC       silk/NLSF_decode.lo
  CC       silk/NSQ.lo
  CC       silk/NSQ_del_dec.lo
  CC       silk/PLC.lo
  CC       silk/shell_coder.lo
  CC       silk/tables_gain.lo
  CC       silk/tables_LTP.lo
  CC       silk/tables_NLSF_CB_NB_MB.lo
  CC       silk/tables_NLSF_CB_WB.lo
  CC       silk/tables_other.lo
  CC       silk/tables_pitch_lag.lo
  CC       silk/tables_pulses_per_block.lo
  CC       silk/VAD.lo
  CC       silk/control_audio_bandwidth.lo
  CC       silk/quant_LTP_gains.lo
  CC       silk/VQ_WMat_EC.lo
  CC       silk/HP_variable_cutoff.lo
  CC       silk/NLSF_encode.lo
  CC       silk/NLSF_VQ.lo
  CC       silk/NLSF_unpack.lo
  CC       silk/NLSF_del_dec_quant.lo
  CC       silk/process_NLSFs.lo
  CC       silk/stereo_LR_to_MS.lo
  CC       silk/stereo_MS_to_LR.lo
  CC       silk/check_control_input.lo
  CC       silk/control_SNR.lo
  CC       silk/init_encoder.lo
  CC       silk/control_codec.lo
  CC       silk/A2NLSF.lo
  CC       silk/ana_filt_bank_1.lo
  CC       silk/biquad_alt.lo
  CC       silk/bwexpander_32.lo
  CC       silk/bwexpander.lo
  CC       silk/debug.lo
  CC       silk/decode_pitch.lo
  CC       silk/inner_prod_aligned.lo
  CC       silk/lin2log.lo
  CC       silk/log2lin.lo
  CC       silk/LPC_analysis_filter.lo
  CC       silk/LPC_inv_pred_gain.lo
  CC       silk/table_LSF_cos.lo
  CC       silk/NLSF2A.lo
  CC       silk/NLSF_stabilize.lo
  CC       silk/NLSF_VQ_weights_laroia.lo
  CC       silk/pitch_est_tables.lo
  CC       silk/resampler.lo
  CC       silk/resampler_down2_3.lo
  CC       silk/resampler_down2.lo
  CC       silk/resampler_private_AR2.lo
  CC       silk/resampler_private_down_FIR.lo
  CC       silk/resampler_private_IIR_FIR.lo
  CC       silk/resampler_private_up2_HQ.lo
  CC       silk/resampler_rom.lo
  CC       silk/sigm_Q15.lo
  CC       silk/sort.lo
  CC       silk/sum_sqr_shift.lo
  CC       silk/stereo_decode_pred.lo
  CC       silk/stereo_encode_pred.lo
  CC       silk/stereo_find_predictor.lo
  CC       silk/stereo_quant_pred.lo
  CC       silk/LPC_fit.lo
  CC       silk/float/apply_sine_window_FLP.lo
  CC       silk/float/corrMatrix_FLP.lo
  CC       silk/float/encode_frame_FLP.lo
  CC       silk/float/find_LPC_FLP.lo
  CC       silk/float/find_LTP_FLP.lo
  CC       silk/float/find_pitch_lags_FLP.lo
  CC       silk/float/find_pred_coefs_FLP.lo
  CC       silk/float/LPC_analysis_filter_FLP.lo
  CC       silk/float/LTP_analysis_filter_FLP.lo
  CC       silk/float/LTP_scale_ctrl_FLP.lo
  CC       silk/float/noise_shape_analysis_FLP.lo
  CC       silk/float/process_gains_FLP.lo
  CC       silk/float/regularize_correlations_FLP.lo
  CC       silk/float/residual_energy_FLP.lo
  CC       silk/float/warped_autocorrelation_FLP.lo
  CC       silk/float/wrappers_FLP.lo
  CC       silk/float/autocorrelation_FLP.lo
  CC       silk/float/burg_modified_FLP.lo
  CC       silk/float/bwexpander_FLP.lo
  CC       silk/float/energy_FLP.lo
  CC       silk/float/inner_product_FLP.lo
  CC       silk/float/k2a_FLP.lo
  CC       silk/float/LPC_inv_pred_gain_FLP.lo
  CC       silk/float/pitch_analysis_core_FLP.lo
  CC       silk/float/scale_copy_vector_FLP.lo
  CC       silk/float/scale_vector_FLP.lo
  CC       silk/float/schur_FLP.lo
  CC       silk/float/sort_FLP.lo
  CC       src/opus.lo
  CC       src/opus_decoder.lo
  CC       src/opus_encoder.lo
  CC       src/opus_multistream.lo
  CC       src/opus_multistream_encoder.lo
  CC       src/opus_multistream_decoder.lo
  CC       src/repacketizer.lo
  CC       src/opus_projection_encoder.lo
  CC       src/opus_projection_decoder.lo
  CC       src/mapping_matrix.lo
  CC       src/analysis.lo
  CC       src/mlp.lo
  CC       src/mlp_data.lo
  CCLD     libopus.la
make[3]: Entering directory '/media/rcampbel/4TB_EXT/rcampbel/project/soundsync/src/utils/opus_vendor/doc'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/media/rcampbel/4TB_EXT/rcampbel/project/soundsync/src/utils/opus_vendor/doc'
make[2]: Leaving directory '/media/rcampbel/4TB_EXT/rcampbel/project/soundsync/src/utils/opus_vendor'
make[1]: Leaving directory '/media/rcampbel/4TB_EXT/rcampbel/project/soundsync/src/utils/opus_vendor'
```

next, cd .. 
ok...

next:
cd ../
emcc -s INITIAL_MEMORY=10MB -s MAXIMUM_MEMORY=10MB -O2 -o opus_wasm.js -s EXPORT_ES6=1 -s MODULARIZE=1 -s SINGLE_FILE=1 -s EXPORT_NAME="Opus" -s USE_ES6_IMPORT_META=0 -s FILESYSTEM=0 -s EXPORTED_RUNTIME_METHODS="['setValue', 'getValue', 'AsciiToString']" -s EXPORTED_FUNCTIONS="['_malloc', '_free', '_opus_decoder_create','_opus_decode_float','_opus_decoder_destroy','_opus_encoder_create','_opus_encoder_destroy','_opus_encode','_opus_strerror']" -s ENVIRONMENT=node,web ./opus_vendor/.libs/libopus.a

'em' should be removed from front of this line.

I tried 3 compilers...

```
rcampbel@Godzilla~/project/soundsync/src/utils $ gcc-9 -s INITIAL_MEMORY=10MB -s MAXIMUM_MEMORY=10MB -O2 -o opus_wasm.js -s EXPORT_ES6=1 -s MODULARIZE=1 -s SINGLE_FILE=1 -s EXPORT_NAME="Opus" -s USE_ES6_IMPORT_META=0 -s FILESYSTEM=0 -s EXPORTED_RUNTIME_METHODS="['setValue', 'getValue', 'AsciiToString']" -s EXPORTED_FUNCTIONS="['_malloc', '_free', '_opus_decoder_create','_opus_decode_float','_opus_decoder_destroy','_opus_encoder_create','_opus_encoder_destroy','_opus_encode','_opus_strerror']" -s ENVIRONMENT=node,web ./opus_vendor/.libs/libopus.a
gcc-9: error: INITIAL_MEMORY=10MB: No such file or directory
gcc-9: error: MAXIMUM_MEMORY=10MB: No such file or directory
gcc-9: error: EXPORT_ES6=1: No such file or directory
gcc-9: error: MODULARIZE=1: No such file or directory
gcc-9: error: SINGLE_FILE=1: No such file or directory
gcc-9: error: EXPORT_NAME=Opus: No such file or directory
gcc-9: error: USE_ES6_IMPORT_META=0: No such file or directory
gcc-9: error: FILESYSTEM=0: No such file or directory
gcc-9: error: EXPORTED_RUNTIME_METHODS=['setValue', 'getValue', 'AsciiToString']: No such file or directory
gcc-9: error: EXPORTED_FUNCTIONS=['_malloc', '_free', '_opus_decoder_create','_opus_decode_float','_opus_decoder_destroy','_opus_encoder_create','_opus_encoder_destroy','_opus_encode','_opus_strerror']: No such file or directory
gcc-9: error: ENVIRONMENT=node,web: No such file or directory
rcampbel@Godzilla~/project/soundsync/src/utils $ cc --version
cc (Ubuntu 4.8.5-4ubuntu8~14.04.2) 4.8.5
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

rcampbel@Godzilla~/project/soundsync/src/utils $ cc -s INITIAL_MEMORY=10MB -s MAXIMUM_MEMORY=10MB -O2 -o opus_wasm.js -s EXPORT_ES6=1 -s MODULARIZE=1 -s SINGLE_FILE=1 -s EXPORT_NAME="Opus" -s USE_ES6_IMPORT_META=0 -s FILESYSTEM=0 -s EXPORTED_RUNTIME_METHODS="['setValue', 'getValue', 'AsciiToString']" -s EXPORTED_FUNCTIONS="['_malloc', '_free', '_opus_decoder_create','_opus_decode_float','_opus_decoder_destroy','_opus_encoder_create','_opus_encoder_destroy','_opus_encode','_opus_strerror']" -s ENVIRONMENT=node,web ./opus_vendor/.libs/libopus.a

cc: error: INITIAL_MEMORY=10MB: No such file or directory
cc: error: MAXIMUM_MEMORY=10MB: No such file or directory
cc: error: EXPORT_ES6=1: No such file or directory
cc: error: MODULARIZE=1: No such file or directory
cc: error: SINGLE_FILE=1: No such file or directory
cc: error: EXPORT_NAME=Opus: No such file or directory
cc: error: USE_ES6_IMPORT_META=0: No such file or directory
cc: error: FILESYSTEM=0: No such file or directory
cc: error: EXPORTED_RUNTIME_METHODS=['setValue', 'getValue', 'AsciiToString']: No such file or directory
cc: error: EXPORTED_FUNCTIONS=['_malloc', '_free', '_opus_decoder_create','_opus_decode_float','_opus_decoder_destroy','_opus_encoder_create','_opus_encoder_destroy','_opus_encode','_opus_strerror']: No such file or directory
cc: error: ENVIRONMENT=node,web: No such file or directory
rcampbel@Godzilla~/project/soundsync/src/utils $ clang --version
clang version 8.0.0 (tags/RELEASE_800/final)
Target: x86_64-unknown-linux-gnu
Thread model: posix
InstalledDir: /usr/bin
rcampbel@Godzilla~/project/soundsync/src/utils $ clang -s INITIAL_MEMORY=10MB -s MAXIMUM_MEMORY=10MB -O2 -o opus_wasm.js -s EXPORT_ES6=1 -s MODULARIZE=1 -s SINGLE_FILE=1 -s EXPORT_NAME="Opus" -s USE_ES6_IMPORT_META=0 -s FILESYSTEM=0 -s EXPORTED_RUNTIME_METHODS="['setValue', 'getValue', 'AsciiToString']" -s EXPORTED_FUNCTIONS="['_malloc', '_free', '_opus_decoder_create','_opus_decode_float','_opus_decoder_destroy','_opus_encoder_create','_opus_encoder_destroy','_opus_encode','_opus_strerror']" -s ENVIRONMENT=node,web ./opus_vendor/.libs/libopus.a
clang-8: error: no such file or directory: 'INITIAL_MEMORY=10MB'
clang-8: error: no such file or directory: 'MAXIMUM_MEMORY=10MB'
clang-8: error: no such file or directory: 'EXPORT_ES6=1'
clang-8: error: no such file or directory: 'MODULARIZE=1'
clang-8: error: no such file or directory: 'SINGLE_FILE=1'
clang-8: error: no such file or directory: 'EXPORT_NAME=Opus'
clang-8: error: no such file or directory: 'USE_ES6_IMPORT_META=0'
clang-8: error: no such file or directory: 'FILESYSTEM=0'
clang-8: error: no such file or directory: 'EXPORTED_RUNTIME_METHODS=['setValue', 'getValue', 'AsciiToString']'
clang-8: error: no such file or directory: 'EXPORTED_FUNCTIONS=['_malloc', '_free', '_opus_decoder_create','_opus_decode_float','_opus_decoder_destroy','_opus_encoder_create','_opus_encoder_destroy','_opus_encode','_opus_strerror']'
clang-8: error: no such file or directory: 'ENVIRONMENT=node,web'
```

So, I tried stripping out the bits thjat were not recognized and get this:
```
rcampbel@Godzilla~/project/soundsync/src/utils $ gcc-9 -s -O2 -o opus_wasm.js ./opus_vendor/.libs/libopus.a
/usr/bin/ld: /usr/lib/debug/usr/lib/x86_64-linux-gnu/crt1.o(.debug_info): relocation 0 has invalid symbol index 11
/usr/bin/ld: /usr/lib/debug/usr/lib/x86_64-linux-gnu/crt1.o(.debug_info): relocation 1 has invalid symbol index 12
/usr/bin/ld: /usr/lib/debug/usr/lib/x86_64-linux-gnu/crt1.o(.debug_info): relocation 2 has invalid symbol index 2
/usr/bin/ld: /usr/lib/debug/usr/lib/x86_64-linux-gnu/crt1.o(.debug_info): relocation 3 has invalid symbol index 2
/usr/bin/ld: /usr/lib/debug/usr/lib/x86_64-linux-gnu/crt1.o(.debug_info): relocation 4 has invalid symbol index 11
/usr/bin/ld: /usr/lib/debug/usr/lib/x86_64-linux-gnu/crt1.o(.debug_info): relocation 5 has invalid symbol index 13
/usr/bin/ld: /usr/lib/debug/usr/lib/x86_64-linux-gnu/crt1.o(.debug_info): relocation 6 has invalid symbol index 13
/usr/bin/ld: /usr/lib/debug/usr/lib/x86_64-linux-gnu/crt1.o(.debug_info): relocation 7 has invalid symbol index 13
/usr/bin/ld: /usr/lib/debug/usr/lib/x86_64-linux-gnu/crt1.o(.debug_info): relocation 8 has invalid symbol index 12
/usr/bin/ld: /usr/lib/debug/usr/lib/x86_64-linux-gnu/crt1.o(.debug_info): relocation 9 has invalid symbol index 13
/usr/bin/ld: /usr/lib/debug/usr/lib/x86_64-linux-gnu/crt1.o(.debug_info): relocation 10 has invalid symbol index 13
/usr/bin/ld: /usr/lib/debug/usr/lib/x86_64-linux-gnu/crt1.o(.debug_info): relocation 11 has invalid symbol index 13
/usr/bin/ld: /usr/lib/debug/usr/lib/x86_64-linux-gnu/crt1.o(.debug_info): relocation 12 has invalid symbol index 13
/usr/bin/ld: /usr/lib/debug/usr/lib/x86_64-linux-gnu/crt1.o(.debug_info): relocation 13 has invalid symbol index 13
/usr/bin/ld: /usr/lib/debug/usr/lib/x86_64-linux-gnu/crt1.o(.debug_info): relocation 14 has invalid symbol index 13
/usr/bin/ld: /usr/lib/debug/usr/lib/x86_64-linux-gnu/crt1.o(.debug_info): relocation 15 has invalid symbol index 13
/usr/bin/ld: /usr/lib/debug/usr/lib/x86_64-linux-gnu/crt1.o(.debug_info): relocation 16 has invalid symbol index 13
/usr/bin/ld: /usr/lib/debug/usr/lib/x86_64-linux-gnu/crt1.o(.debug_info): relocation 17 has invalid symbol index 13
/usr/bin/ld: /usr/lib/debug/usr/lib/x86_64-linux-gnu/crt1.o(.debug_info): relocation 18 has invalid symbol index 13
/usr/bin/ld: /usr/lib/debug/usr/lib/x86_64-linux-gnu/crt1.o(.debug_info): relocation 19 has invalid symbol index 21
/usr/bin/ld: /usr/lib/debug/usr/lib/x86_64-linux-gnu/crt1.o(.debug_line): relocation 0 has invalid symbol index 2
/usr/lib/gcc/x86_64-linux-gnu/9/../../../x86_64-linux-gnu/crt1.o: In function `_start':
(.text+0x20): undefined reference to `main'
collect2: error: ld returned 1 exit status
```

Then I tried installing your .deb file and it installs, and it looked like it succeeded but I saw nothing installed,

so I run 'ar x soundsync_0.1.0_amd64.deb` and then 'tar Jxvf data.tar.xz'

now I can see the binaries that should have gone into /opt/Soundsync
I've moved those to /opt/Soundsync and now I can run soundsync from there and open up http://localhost:6512/

Hope this information is helpful

**Comment by geekuillaume** [comment]
The `em` prefix before `configure`, `make` and `cc` are needed to use the Emscripten version of these tools. The Opus bindings are compiled for Webassembly to use them from NodeJs. But anyway, these instructions are just necessary if you want to change some specific options for Opus, I already included a compiled Wasm Opus Bindings in the repository [here](https://github.com/geekuillaume/soundsync/blob/master/src/utils/opus_wasm.js). 

I've just tried installing it on Debian from the provided `.deb` and everything is working fine. When you installed it nothing was in `/opt/Soundsync`?
